src/main/resources/templates/metadata-management.html

```diff
@@ -73,12 +83,12 @@ <h4>Metadata Record Form</h4>
                 $("#formModal").modal('show');
             },
             updateOperation: function (rowColumnvalue) {
-                $.getJSON("/jsonSchemas/metadataRecordForCreate.json", function (dataModel) {
-                    $.getJSON("/jsonSchemas/uiFormmetadataRecordForCreate.json", function (uiForm) {
+                $.getJSON("/jsonSchemas/metadataRecord.json", function (dataModel) {
```

Comment:
Please use variables instead of writing fixed paths completely over and over again. (if this is possible)
e.g. "/jsonSchemas" -> pathToSchemas